gulp/config.js

```diff
@@ -0,0 +1,9 @@
+var path          = require('path');
+var environments  = require('gulp-environments');
+
+module.exports = {
+    src     : 'src/AppBundle/Resources/assets',
+    dest    : 'src/AppBundle/Resources/public',
+    dev     : environments.development,
+    prod    : environments.production
+};
```

Comment:
New lines are missing from files. Git prefers having an empty new line at the end of each file.